Description: Temporarily disable Arbitrary derive for zopfli::Options
 It uses NonZeroU64, but proptest added support only in 1.2, while we have 1.0
Forwarded: not-needed
Last-Update: 2024-06-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -90 +90 @@
-#[cfg_attr(all(test, feature = "std"), derive(proptest_derive::Arbitrary))]
+//#[cfg_attr(all(test, feature = "std"), derive(proptest_derive::Arbitrary))]
@@ -98,6 +98,6 @@
-    #[cfg_attr(
-        all(test, feature = "std"),
-        proptest(
-            strategy = "(1..=10u64).prop_map(|iteration_count| NonZeroU64::new(iteration_count).unwrap())"
-        )
-    )]
+    //#[cfg_attr(
+    //    all(test, feature = "std"),
+    //    proptest(
+    //        strategy = "(1..=10u64).prop_map(|iteration_count| NonZeroU64::new(iteration_count).unwrap())"
+    //    )
+    //)]
@@ -205 +205 @@
-            options: Options,
+            //options: Options,
@@ -211 +211 @@
-            let mut encoder = DeflateEncoder::new(options, btype, &mut compressed_data);
+            let mut encoder = DeflateEncoder::new(Default::default(), btype, &mut compressed_data);
